spec/features/hiring_staff_can_copy_a_vacancy_spec.rb

```diff
@@ -153,31 +153,31 @@
       let(:new_attributes) { { publish_on: nil } }
 
       it 'shows an error' do
-        expect(page).to have_content("Publish on can't be blank")
+        expect(page).to have_content('Enter the date the role will be listed')
       end
     end
 
     context 'when publish on date is in the past' do
       let(:new_attributes) { { publish_on: 1.day.ago } }
 
       it 'shows an error' do
-        expect(page).to have_content("Publish on can't be before today")
+        expect(page).to have_content('Date role will be listed must be in the future')
```

Comment:
This should be changed to `Date role will be listed must be either today or a date in the future` as the above error is not representative of how the field behaves

Reply:
I would argue with this. Even if it's today it must be in the future, therefore I'm happy with this message

Reply:
Sorry I missed that its an ongoing discussion, I updated this in 45aac81